src/Slider/slider-style.scss

```diff
@@ -107,13 +107,13 @@ $box-shadow: rgba(85, 85, 85, 0.4) 0 2px 5px;
 
 .form-group.custom-slider {
     padding-right: 5px;
-    label.custom-label:not(.form-check-label):not(.custom-checkbox):not(.custom-radio):not(.custom-control-label) {
-        display: block;
-        margin-bottom: 4px;
-        line-height: 1;
-        font-weight: normal;
-        font-size: 12px;
-    }
+    // label.custom-label:not(.form-check-label):not(.custom-checkbox):not(.custom-radio):not(.custom-control-label) {
```

Comment:
Why is this commented? if you don't need it just remove it

Reply:
resolved in commit https://github.com/sebgroup/react-components/pull/29/commits/2b668852b23d63790e2f37e7a82001a8673b8ad5